tests/Backend/FileWorkspace/WorkspacesLoadTest.php

```diff
@@ -1180,4 +1180,159 @@ private function assertManifest(Abs $backend, $destination)
+                    "dataFileId" => $file1Id,
+                    "destination" => "fileLanguages",
+                ],
+                [
+                    'dataFileId' => $file2Id,
+                    'destination' => 'fileUsers',
+                ],
+            ],
+        ];
+
+        $workspaces->loadWorkspaceData($workspace['id'], $options);
+
+        $data = $backend->fetchAll('tableLanguages', ["id","name"], false, false);
+        $this->assertCount(6, $data);
+        $this->assertArrayEqualsSorted(
+            $this->_readCsv($table1Csv),
+            $data,
+            0
+        );
+
+        $data = $backend->fetchAll('tableUsers', ["id","name", "city", "sex"], false, false);
+        $this->assertArrayEqualsSorted(
+            $this->_readCsv($table2Csv),
+            $data,
+            0
+        );
+
+        $data = $backend->fetchAll('fileLanguages', ["id", "name"], true, true, false);
+        $this->assertCount(6, $data);
+        $this->assertArrayEqualsSorted(
+            $this->_readCsv($file1Csv),
+            $data,
+            0
+        );
+
+        $data = $backend->fetchAll('fileUsers', ["id","name", "city", "sex"], true, true, false);
+        $this->assertArrayEqualsSorted(
+            $this->_readCsv($file2Csv),
+            $data,
+            0
+        );
+
+        $table3Csv = __DIR__ . '/../../_data/languages.more-rows.csv';
+        $table3Id = $this->_client->createTable(
+            $this->getTestBucketId(self::STAGE_IN),
+            'languagesMoreRows',
+            new CsvFile($table3Csv)
+        );
+
+        $file3Csv = __DIR__ . '/../../_data/languages.more-rows.csv';
+        $file3Id = $this->_client->uploadFile(
+            (new CsvFile($file3Csv))->getPathname(),
+            (new FileUploadOptions())
+                ->setNotify(false)
+                ->setIsPublic(false)
+                ->setCompress(true)
+                ->setTags(['test-file-2'])
+        );
+
+        $options = [
+            'preserve' => true,
+            'input' => [
+                [
+                    'source' => $table3Id,
+                    'destination' => 'tableLanguages',
+                    'overwrite' => true,
+                ],
+                [
+                    "dataFileId" => $file3Id,
+                    "destination" => "fileLanguages",
+                    'overwrite' => true,
+                ],
+            ],
+        ];
+
+        // load should overwrite existing tables with new values
+        $workspaces->loadWorkspaceData($workspace['id'], $options);
+
+        // test load from table overwrite existing data instead of add new data
+        $data = $backend->fetchAll('tableLanguages', ["id", "name"], false, false);
+        $this->assertCount(8, $data);
+
+        // test load from file overwrite existing data instead of add new data
+        $data = $backend->fetchAll('fileLanguages', ["id", "name"], true, true, false);
+        $this->assertCount(8, $data);
```

Comment:
Ked som to chcel testovat este pomocou `assertArrayEqualsSorted` tak to padalo na tom ze to nebolo zoradene jak to csvcko. Neviem ci som nieco neprehliadok. Ked tak este upravim po review

Reply:
Jo synapse to neexportuje seřazené.

Reply:
Přihodil sem si tam ten assert a prošlo mě to.

Reply:
@zajca pushnes to prosim ? Ja som tam tiez nieco pridaval a preslo mi to a pak zas ne, ze mi to vzdy zoradilo inac.

Reply:
Jak dokopnu paralelní uploady tak to pushnu zkusím to pak ještě znova jestli to třeba nebude náhodně failovat.

Reply:
Jo ja som zacal upravovat tu vec co si pisal v tom main PR tak to este skusim aj ja.